Packs/SecneurXThreatFeeds/Integrations/SecneurXThreatFeeds/SecneurXThreatFeeds.py

```diff
@@ -10,16 +10,71 @@
 
 # Disable insecure warnings
 requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
-SNX_ID_KEY = "id"
-SNX_PATTERN_KEY = "pattern"
-SNX_OBJECTS_KEY = "objects"
-SNX_CREATED_KEY = "created"
-SNX_MODIFIED_KEY = "modified"
-SNX_DESCRIPTIONS_KEY = "description"
-SNX_FEED_KEY = "SecneurX Threat Feeds"
 DEMISO_DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
 
 
+SNX_TYPES_TO_DEMISTO_TYPES = {
+    'url:value': FeedIndicatorType.URL,
+    'md5:value': FeedIndicatorType.File,
+    'sha-1:value': FeedIndicatorType.File,
+    'sha-256:value': FeedIndicatorType.File,
+    'ipv4-addr:value': FeedIndicatorType.IP,
+    'domain:value': FeedIndicatorType.Domain,
+    'ipv6-addr:value': FeedIndicatorType.IPv6,
+    'email-addr:value': FeedIndicatorType.Email,
+    'domain-name:value': FeedIndicatorType.Domain,
+    'file:hashes.MD5': FeedIndicatorType.File
+}
+
+STIX_INTEL_TYPE_TO_DEMISTO_TYPES = {
+    'campaign': ThreatIntel.ObjectsNames.CAMPAIGN,
+    'attack-pattern': ThreatIntel.ObjectsNames.ATTACK_PATTERN,
+    'report': ThreatIntel.ObjectsNames.REPORT,
+    'malware': ThreatIntel.ObjectsNames.MALWARE,
+    'course-of-action': ThreatIntel.ObjectsNames.COURSE_OF_ACTION,
+    'intrusion-set': ThreatIntel.ObjectsNames.INTRUSION_SET,
+    'tool': ThreatIntel.ObjectsNames.TOOL,
+    'threat-actor': ThreatIntel.ObjectsNames.THREAT_ACTOR,
+    'infrastructure': ThreatIntel.ObjectsNames.INFRASTRUCTURE,
+}
+
+XSOAR_RELATIONSHIP_TYPES = EntityRelationship.Relationships.RELATIONSHIPS_NAMES.keys()
+
+FILE_HASHES_KEY_LIST = [
+    'file:hashes.MD5',
+    'file:hashes.SHA-1',
+    'file:hashes.SHA-256'
```

Comment:
What do you need it for? They are of no use.

Reply:
We added for future support of STIX 2.1 format. We have removed in the latest commit